contents/ruby/Rails.md

```diff
@@ -0,0 +1,53 @@
+<frontmatter>
+  title: Introduction to Ruby on Rails
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Ruby on Rails
+
+Authors: [Chattoraj Ayush](https://github.com/AyushChatto)
+
+## Ruby on Rails overview
+
+**Ruby on Rails** is an **easy-to-learn** framework that allows you to develop powerful web applications **quickly** and with as little effort as possible. It was developed in the early 2000s by Danish Programmer David Heinemeier Hansson, or 'DHH'. 
+
+Ruby on Rails, or simply Rails, gained widespread popularity in the 2000s, and continues to be one of the most popular web-frameworks even to this day, because of how quickly it allows an easily scalable minimum viable product to be built, making it the perfect choice for startups. This is owed in large part to the Rails Doctrines of **Optimising for Programmer Happiness** and **Convention over Configuration**.
+
+> ... I created Rails for me. To make me smile, first and foremost. Its utility was to many degrees subservient to its ability to make me enjoy my life more. To enrich my daily toil of wrangling requirements and requests for web information systems.
+>
+> ... One of the early productivity mottos of Rails went: “You’re not a beautiful and unique snowflake”. It postulated that by giving up vain individuality, you can leapfrog the toils of mundane decisions, and make faster progress in areas that really matter.
+>
+> *David Heinemeier Hansson,* __The Rails Doctrine__
+
+Many popular and very technically mature websites continue to use Rails. These include: GitHub, Airbnb, Twitter, Hulu, Shopify, and Twitch, among others.
+
+## Getting Started
```

Comment:
As mentioned, I think it's better to shift this section to (near) the end.

Reply:
Gotcha, it does flow better that way :)